docs/troubleshooting.md

```diff
@@ -0,0 +1,102 @@
+# Common issues
```

Comment:
nit: almost all the issues here are for podman, might be good to call that out explicitly... maybe something like

```suggestion
# Common issues

## Podman
```

wdyt?

Reply:
I decided to structure it by error message instead, most issues are podman-related now, but I expect that similar ones might start coming for other providers later